templates/default/document/documentedit.html

```diff
@@ -151,7 +151,7 @@ <H1>{$layout.pagetitle}</H1>
 		</TD>
 	</TR>
 *}
-	{if !$document.closed && ($document.docrights & $smarty.const.DOCRIGHT_CONFIRM)}
+	{if $document.docrights && $smarty.const.DOCRIGHT_CONFIRM}
 	<TR>
```

Comment:
Wiesze, że to spowoduje, że będziesz mógł ponownie zatwierdzać dokumenty już zatwierdzone?
Chciałeś uzyskać możliwość cofnięcia zatwierdzenia, gdy dokument jest zatwierdzony i jednocześnie użytkownik ma do tego prawo bez zmiany zachowania samych zatwierdzeń?

Reply:
> Wiesze, że to spowoduje, że będziesz mógł ponownie zatwierdzać dokumenty już zatwierdzone?

Tak.

> Chciałeś uzyskać możliwość cofnięcia zatwierdzenia, gdy dokument jest zatwierdzony i jednocześnie użytkownik ma do tego prawo bez zmiany zachowania samych zatwierdzeń?

Tak ale po tym co napisałeś stwierdzam, że faktycznie możliwość odzatwierdzenia "odzatwierdzenie" powinien mieć tylko użytkownik o pełnych uprawnieniach a nie tylko Ci, którzy decydują o zatwierdzaniu. Historia zatwierdzeń nie jest potrzebna.